dfx.json

```diff
@@ -20,10 +20,6 @@
     "fake_btc" : {
       "main": "src/fake_btc/token.mo",
       "type": "motoko"
-    },
-    "mint_token": {
-      "main": "src/mint_token/token.mo",
-      "type": "motoko"
```

Comment:
move it back please
if you need to update/deploy only your canister - you could specify it in cli

Reply:
update restriction

Reply:
I create mint_token inside the protocol canister with init function.
So, it should not be deployed independently.